src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write your code here
+const tableData = createDataArr('tr');
+const table = () => {
+  const newTable = [];
+
+  for (let i = 0; i < tableData.length; i++) {
+    (i === 0 || i === tableData.length - 1)
+      ? newTable.push(getRowTexts('th', i))
+      : newTable.push(getRowTexts('td', i));
+  }
+
+  return newTable;
+};
+
+for (let i = 0; i < table().length; i++) {
```

Comment:
DRY and not obvious. 
According your variable `const newTable = [];` you need to create the table once, instead you create it every time you call the function.

`table()` - move to variable

Reply:
Also I did use var for table and then made it as a func)))
Again I need more practice to choose the best way to implement )